### DISCUSSION TITLE
v57.0

### DISCUSSION THREAD

**Discussion by westnordost** [streetcomplete/StreetComplete#5519] [Release Discussions]
Take cover! For you don't want to be squashed by this m-m-mega phat update, or do you?

### Buildings Overlay (#5461)

<img src="https://github.com/streetcomplete/StreetComplete/assets/4661658/f9e6417d-5344-4e1d-9d80-7d1ce8533d84" width="300">

The new Buildings overlay now lets you comprehensively view and edit building types all around town, color-coded for your convenience!
Conversely, the building type quest is disabled by default now as it was really spammy, don't you think?

(You'll be amazed and/or appalled how many buildings are currently actually tagged wrong or imprecise! 😅)

### Things Overlay (#4912), thanks @matkoniecz

<img  src="https://github.com/streetcomplete/StreetComplete/assets/4661658/0f971035-06a5-474b-b410-41fcc5d4f7da" width="300"> 

The new Things overlay allows you to map all those small map features like benches, bicycle parkings, roadside trees, ATMs and other street furniture in general.

It complements the Places Overlay which has been renamed from Shop overlay because it now also lets you map places that are not shops - like hotels, hospitals, schools and so forth. (#5152)

### New Quests

- The crossing type quest has been split up into two quests: One that asks for markings, the other for traffic signals (#5471, #5476)

### Quest and Overlay Enhancements and Fixes

- Lit and Surface Overlay: now each have a button to apply the last answer (#4741)
- Measuring widths and heights: [StreetMeasure](https://play.google.com/store/apps/details?id=de.westnordost.streetmeasure) doesn't work for you? In that case you'll be offered to quickly disable these AR measurement quests now (#4849)
- Max Height: Don't prompt to estimate height when there is no sign (#5458)
- Opening Hours: Also ask for places whose already set opening hours likely contain mistakes (#5463)
- BBQ fuel: Add "gas" answer (#5495), by @k-yle
- Steps: Don't ask about ramps and tactile paving for hiking steps, by @matkoniecz
- Fire Hydrant Diameter: Disable for Austria again (#5470)

### Other

- Fix a (shockingly common) crash (#5498)
- Fix issues with doing edits while data is being downloaded (#4258)
- Add some interesting links as achievement rewards (#5466)
- Translation to Amharic has been disabled, it was not maintained for over a year
- Use new method to upload and download data as mandated by Google Play policy. On Android 13 and above, there's no notification for continuing to sync in the background anymore. (#5492)
- Other small enhancements, performance and wording improvements (#5468, #5154, #5474...), thanks @Jean-BaptisteC, @matkoniecz, @FloEdelmann

<hr /><em>This discussion was created from the release <a href='https://github.com/streetcomplete/StreetComplete/releases/tag/v57.0'>v57.0</a>.</em>

**Comment by Matthieu-LAURENT39** [comment]
The things overlay is a real game changer! Great work on this update!
